### DISCUSSION TITLE
1.7.0

### DISCUSSION THREAD

**Discussion by chucker** [chucker/Mastonaut#120] [Announcements]
## Features

* You can now **customize the font face and size** in Preferences. (#83) By default, the font of a focused status is a bit larger than others, so you can set both separately.
* For the focused status, you can see interaction statistics (how many replies, boosts, and favorites did it receive).
* In addition to letting you _bookmark_ a hashtag as before, you can now **follow a hashtag**. Posts mentioning the tag will then appear in your regular home timeline (not just in Mastonaut, but any client), just like _users_ you're following.

## Fixes

* In 1.6.0, you couldn't hit return when composing to insert a line break. (#103)
* Since 1.5.0, the <kbd>cmd</kbd>-<kbd>c</kbd> wouldn't copy text any more. (#62)
* 1.6.0 could crash when typing in the search box. (#108)
* There's been a crash when loading the time line, due to pull-to-refresh. This is hopefully fixed now. (#63, #95)

<hr /><em>This discussion was created from the release <a href='https://github.com/chucker/Mastonaut/releases/tag/app-1.7.0'>1.7.0</a>.</em>

**Comment by akibasocial** [comment]
Thank you!
